File: HrmsAngular/src/app/models/salary-process/emp-salary-structure.model.ts
import { SalaryStructureView } from "./salary-structure-view.model";

export class EmpSalaryStructure {
     id :number;
     companyID :number;
     empID :number;
     empCode :string;
     structureID :number;
     amount :number;
    empadditionModel:SalaryStructureView[];
    empdeductionModel:SalaryStructureView[];
}

File: HrmsWebApiCore/WebApiCore/Models/SalaryProcess/EmpSalaryStructureModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using WebApiCore.ViewModels.SalaryProcess;

namespace WebApiCore.Models.SalaryProcess
{
    public class EmpSalaryStructureModel
    {
        public int ID { get; set; }
        public int CompanyID { get; set; }
        public int EmpID { get; set; }
        public string EmpCode { get; set; }
        public int StructureID { get; set; }
        public decimal Amount { get; set; }
        public List<SalaryStructureViewModel> EmpAdditionModel { get; set; }
        public List<SalaryStructureViewModel> EmpDeductionModel { get; set; }
    }
}
